src/main/java/org/apache/skywalking/banyandb/v1/client/BanyanDBClient.java

```diff
@@ -314,6 +317,38 @@ public void define(Measure measure) throws BanyanDBException {
         MetadataCache.INSTANCE.register(measure);
     }
 
+    /**
+     * Create or update the property
+     *
+     * @param property the property to be stored in the BanyanBD
+     */
+    public void save(Property property) throws BanyanDBException {
```

Comment:
I think you miss `delete`, is this on purpose?

Reply:
I don't think so. He might miss it out.